src/input/event/value.rs

```diff
@@ -32,6 +32,12 @@ pub enum InputValue {
         y: Option<f64>,
         z: Option<f64>,
     },
+    TouchValue {
+        index: u64,
+        pressed: bool,
```

Comment:
I think we should rename this to `is_touching` instead of `pressed`, so we don't confuse it with a touchpad "press" (which would be like clicking the touchpad on the Steam Deck)